src/ASM/LR/ASMu2Dmx.C

```diff
@@ -335,10 +337,18 @@ bool ASMu2Dmx::integrate (Integrand& integrand,
           fe.v = gpar[1][j];
 
           // Compute basis function derivatives at current integration point
-          std::vector<Go::BasisDerivsSf> splinex(m_basis.size());
-          for (size_t b=0; b < m_basis.size(); ++b) {
-            m_basis[b]->computeBasis(fe.u, fe.v, splinex[b], els[b]-1);
-            SplineUtils::extractBasis(splinex[b],fe.basis(b+1),dNxdu[b]);
+          if (use2ndDer) {
+            for (size_t b = 0; b < m_basis.size(); ++b) {
+              Go::BasisDerivsSf2 spline;
+              m_basis[b]->computeBasis(fe.u,fe.v,spline,els[b]-1);
+              SplineUtils::extractBasis(spline,fe.basis(b+1),dNxdu[b],d2Nxdu2[b]);
+            }
+          } else {
+            for (size_t b=0; b < m_basis.size(); ++b) {
+              Go::BasisDerivsSf spline;
+              m_basis[b]->computeBasis(fe.u, fe.v, spline, els[b]-1);
+              SplineUtils::extractBasis(spline,fe.basis(b+1),dNxdu[b]);
+            }
           }
 
```

Comment:
no, but my mental parser fails. my dislike for the insistence on not using braces for multi-line blocks is huge and i cannot follow it while coding because it slows me down a lot. and i sometime fails at applying the postprocessing step of removing them.